tests/metrics/map_mem.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  The objective of this tool is to report the memory map of a process in
+#  a JSON file
+
+declare PROC="$1"
+declare NUM_SAMPLES=10
```

Comment:
I made some experiments during this script, one of them was to collect different quantity of samples (10, 100, ...500 etc), I noticed that the value of the variance does not affect the values, I got very similar values taking 10 samples and N samples (N > 10  and N < 500), if you consider that it is a good idea to add support for changing this value, I can add it, what do you think?